src/docs/Components/Cards/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Cards'
+order: 1
+description: 'A card is a graphic element that structures a group of related informations that must be understood on its own. It visually looks like a playing card.'
+status:
+  sketch: 'wip'
+  scss: 'wip'
+---
+
+> A card is a graphic element that structures a group of related informations that must be understood on its own. It visually looks like a playing card.
+
+## Usages
+
+A card is mostly a visualisation of the page that is to come behind the card itself.
+
+-- Add a preview of the two card versions next to each other. --
+
+### Outlined version
+
+Outlined cards are used to avoid flotting effects, for exemple when you create parallax or depth effects with differents size of cards.
+
+<Highlight theme="tips" title="Good to know">
+
+This version is best when you have to display few quantity of cards.
+
+</Highlight>
+
+
+### Unboxed version
+
+Unboxed versions are used when you have to display a large quantity of cards, and avoid a graphic complexity because of the outline repetition.
```

Comment:
The unboxed version is best used when you have a large amount of cards to display to avoid graphic complexity due to the repetition of the outline.